xmlsec.go

```diff
@@ -134,3 +134,11 @@ func Decrypt(xml string, privateKeyPath string) ([]byte, error) {
 func deleteTempFile(filename string) {
 	_ = os.Remove(filename)
 }
+
+func LoadXml(certPath string) (string, error) {
```

Comment:
Let's keep this function in case if we have a Request/Response and we need to test/validate somethig with it